ocean/periodic_planar/2.5km/default_gotm/config_analysis.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<config case="analysis">
+	<add_link source="../forward/output.nc" dest="output.nc"/>
+	<add_link source="../plot_profile.py" dest="plot_profile.py"/>
+	<copy_file source_path="script_test_dir" source="plot_profile.py" dest="plot_profile.py"/>
```

Comment:
I think this should be linked only, not copied and it should be linked inside the case directory, not the test directory.

Reply:
```suggestion
	<add_link source_path="script_test_dir" source="plot_profile.py" dest="plot_profile.py"/>
```